src/app/shipping/itemsRequireShipping.ts

```diff
@@ -0,0 +1,16 @@
+import { Cart, StoreConfig } from '@bigcommerce/checkout-sdk';
+
+const itemsRequireShipping = (cart: Cart | undefined, config: StoreConfig | undefined) => {
+    if (cart) {
+        const physicalItems = cart.lineItems.physicalItems.some(lineItem => lineItem.isShippingRequired);
+        if (config && config.checkoutSettings.features['CHECKOUT-4936.enable_custom_item_shipping'] && cart.lineItems.customItems) {
+            return cart.lineItems.customItems.length > 0 || physicalItems;
```

Comment:
instead of having `|| physicalItems;` you can simplify the logic by returning early

```js
if (!cart) {
  return false;
}

if (cart.lineItems.physicalItems.some(lineItem => lineItem.isShippingRequired)) {
  return true;
}

if (config && config.checkoutSettings.features['CHECKOUT-4936.enable_custom_item_shipping'] && cart.lineItems.customItems) {
  return cart.lineItems.customItems.length > 0
}

return false;